### PR TITLE
🌐 Add Japanese translation for `docs/ja/docs/tutorial/extra-data-types.md`

### PR DESCRIPTION
**Pull request by SwftAlpc**
This PR translates the tutorial/extra-data-types.md to Japanese.
Please review tutorial/extra-data-types.md. 

~~Issue #1571 generates a new directory to support Japanese.
PR #1656 generates a new directory for tutorial.
I will fix dependency after #1571 & #1656 are merged.~~

### CODE REVIEW

**Review by tokusumi — CHANGES_REQUESTED** [review]

**Review by tokusumi — APPROVED** [review]
LGTM

File: docs/ja/docs/tutorial/extra-data-types.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    * 生成されたスキーマは`str`で`binary`の「フォーマット」持つことを指定します。
+* `Decimal`:
+    * Pythonの標準的な`Decimal`です。
+    * リクエストやレスポンスでは`float`と同じように扱います。
+
+## 例
+
+ここでは、上記の型のいくつかを使用したパラメータを持つ*path operation*の例を示します。
+
+```Python hl_lines="1 2 11 12 13 14 15"
```
[the remark this answers is not in the document]
Comment (rkbeatss) [reply docs/ja/docs/tutorial/extra-data-types.md]: `hl_lines` supports ranges (i.e, 12-16) and would be more readable / maintainable.  issue for reference: #1859

File: docs/ja/docs/tutorial/extra-data-types.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    * 多くのデータベースやシステムで共通のIDとして使用される、標準的な「ユニバーサルにユニークな識別子」です。
+    * リクエストとレスポンスでは`str`として表現されます。
+* `datetime.datetime`:
+    * Pythonの`datetime.datetime`です。
+    * リクエストとレスポンスはISO 8601形式の`str`で表現されます: `2008-09-15T15:53:00+05:00`
+* `datetime.date`:
+    * Pythonの`datetime.date`です。
+    * リクエストとレスポンスはISO 8601形式の`str`で表現されます: `2008-09-15`
+* `datetime.time`:
+    * Pythonの`datetime.time`.
```
Comment (delhi09) [inline docs/ja/docs/tutorial/extra-data-types.md:34]: "です。" is better at the view of the consistency.

File: docs/ja/docs/tutorial/extra-data-types.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        * リクエストでは、リストが読み込まれ、重複を排除して`set`に変換されます。
+        * レスポンスでは`set`が`list`に変換されます。
+        * 生成されたスキーマは`set`の値が一意であることを指定します（JSON Schemaの`uniqueItems`を使用します）。
+* `bytes`:
+    * Pythonの標準的な`bytes`です。
+    * リクエストとレスポンスでは`str`として扱われます。
+    * 生成されたスキーマは`str`で`binary`の「フォーマット」持つことを指定します。
+* `Decimal`:
+    * Pythonの標準的な`Decimal`です。
+    * リクエストやレスポンスでは`float`と同じように扱います。
```
Comment (delhi09) [inline docs/ja/docs/tutorial/extra-data-types.md:51]: "扱われます。" is better at the view of the consistency.

**Review by delhi09 — COMMENTED** [review]
It looks mostly good.
I pointed out some details.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 549e72092b7feeff14853e3463b536269c24b8bb at: https://6499cb481d7654047104dee1--fastapi.netlify.app

**Comment by alejsdev** [comment]
Thanks for your help @SwftAlpc and @tokusumi 🤓

**Comment by alejsdev** [comment]
Thanks for your help @SwftAlpc and your review @tokusumi 🤓🚀✨

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9231c7983b59c12edb280b23660d07834568d866 at: https://fb5c19c5.fastapitiangolo.pages.dev

**Comment by alejsdev** [comment]
Thank you! @nearnear 😃 

And everyone else for the reviews 🚀✨

**Comment by alejsdev** [comment]
Thank you! @SwftAlpc 😃 

And everyone else for the reviews 🚀✨

**Pull request merged**